### PR TITLE
Prevent imagestream dsl step from continuing if imagestream not present

### PR DESCRIPTION
**Pull request by csrwng**
When using the imagestream dsl step, the jenkins job was not immediately aborting when the imagestream/tag could not be retrieved. (This was not the case with the non-dsl jobs)

This change makes it possible to abort the job while allowing the parent SCMStep to save the state information about the SCM in the job.

### CODE REVIEW

**Comment by csrwng** [comment]
@gabemontero ptal

File: src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftImageStreams.java [hunk 1]
```diff
@@ -5,10 +5,7 @@
 import com.openshift.restclient.IClient;
 import com.openshift.restclient.ResourceKind;
 import com.openshift.restclient.model.IImageStream;
-import hudson.EnvVars;
-import hudson.Extension;
-import hudson.FilePath;
-import hudson.Launcher;
+import hudson.*;
```
Comment (gabemontero) [inline src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftImageStreams.java]: don't know if you did this or an IDE you are using, but I at least think explicitly listing each class is better than using the package wildcard.

thoughts?

**Review by gabemontero — COMMENTED** [review]
one nit but otherwise IGTM

I'll wait until the current release cut completes, then if we've mitigated the nit comment, I'll merge this and start a new release cut.

File: src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftImageStreams.java (hunk 1 above)
Comment (csrwng) [reply src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftImageStreams.java]: I agree. IntelliJ is trying to be too intelligent. Will update.

**Comment by gabemontero** [comment]
btw the release publish process is done.  v1.0.24 is available.  I'll crank out the next version with this PR once we mitigate the nit comment.

**Comment by csrwng** [comment]
man, that was quick, thx :)

**Comment by gabemontero** [comment]
On Thursday, October 20, 2016, Cesar Wong notifications@github.com wrote:

> man, that was quick, thx :)

Likewise thx :-)

**Pull request merged**